examples/braket_features/Simulating_Advanced_OpenQASM_Programs_with_the_Local_Simulator.ipynb

```diff
@@ -0,0 +1,726 @@
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Subroutines\n",
+    "\n",
+    "The `LocalSimulator` supports creating subroutines to compartmentalize pieces of your code.\n",
+    "They can support both classical and quantum computation.\n",
+    "\n",
+    "For documentation around subroutine, see\n",
+    "https://openqasm.com/language/subroutines.html"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Measurement counts: Counter({'1': 100})\n"
+     ]
+    }
+   ],
+   "source": [
+    "qasm_string = \"\"\"\n",
+    "const int[8] n = 4;\n",
+    "input bit[n] x;\n",
+    "\n",
+    "qubit q;\n",
+    "\n",
+    "def parity(bit[n] cin) -> bit {\n",
+    "  bit c = false;\n",
+    "  for int[8] i in [0: n - 1] {\n",
+    "    c ^= cin[i];\n",
+    "  }\n",
+    "  return c;\n",
+    "}\n",
+    "\n",
+    "if(parity(x)) {\n",
+    "    x q;\n",
+    "} else {\n",
+    "    i q;\n",
+    "}\n",
+    "\"\"\"\n",
+    "qasm_program = Program(source=qasm_string, inputs={\"x\": \"1011\"})\n",
+    "\n",
+    "result = device.run(qasm_program, shots=100).result()\n",
+    "print(f\"Measurement counts: {result.measurement_counts}\")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Measurement counts: Counter({'0': 88, '1': 12})\n"
+     ]
+    }
+   ],
+   "source": [
+    "qasm_string = \"\"\"\n",
+    "qubit q;\n",
+    "\n",
+    "def sum(const array[int[8], #dim = 1] arr) -> int {\n",
+    "    int size = sizeof(arr);\n",
+    "    int x = 0;\n",
+    "    for int i in [0:size - 1] {\n",
+    "        x += arr[i];\n",
+    "    }\n",
+    "    return x;\n",
+    "}\n",
+    "\n",
+    "array[int, 10] arr = {9, 3, 6, 2, 2, 4, 3, 1, 12, 7};\n",
+    "int s = sum(arr);\n",
+    "rx(s*π/4) q;\n",
+    "\"\"\"\n",
+    "qasm_program = Program(source=qasm_string)\n",
+    "\n",
+    "result = device.run(qasm_program, shots=100).result()\n",
+    "print(f\"Measurement counts: {result.measurement_counts}\")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 2
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython2",
+   "version": "2.7.6"
+  }
+ },
+ "nbformat": 4,
```

Comment:
Similarly, this should have conda_braket as the kernel as we have for other notebooks.

Reply:
Fixed by exporting notebook from NBI